tools/ubuntu-setup/pip.sh

```diff
@@ -1,4 +1,3 @@
 sudo apt-get install -y python-pip
 sudo pip install jsonschema
 sudo pip install argcomplete
-sudo pip install docker-py
```

Comment:
It's essentially a dependecy of `ansible` in our build, hence it makes sense to install it in conjunction with ansible.